File: 2st/Yeonwoo/OX퀴즈.py
def solution(quiz):
    answer = []
    temp = 0
    for i in quiz:
        a, op, b, ev, c = i.split()
        if op == '+':
            temp = int(a)+int(b)
            if temp == int(c):
                answer.append('O')
            else:
                answer.append('X')
        else:
            temp = int(a)-int(b)
            if temp == int(c):
                answer.append('O')
            else:
                answer.append('X')
                
        
    return answer

File: 4st/Eunseong/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        for (int i = 0; i < answer.length; i++) {
            String temp[] = quiz[i].split(" ");
            if (temp[1].equals("+")) {
                if (Integer.parseInt(temp[0]) + Integer.parseInt(temp[2]) == Integer.parseInt(temp[4])) answer[i] = "O";
                else answer[i] = "X";
            } else {
                if (Integer.parseInt(temp[0]) - Integer.parseInt(temp[2]) == Integer.parseInt(temp[4])) answer[i] = "O";
                else answer[i] = "X";
            }
        }
        return answer;
    }
}